debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
Is there something akin to %exclude that is used in pbspro.spec?

Why don't we just use alien to convert an RPM to a .deb package? This seems like a lot of extra work, and something else we're going to have to maintain in the future.